R/model.R

```diff
@@ -548,12 +563,16 @@ model_exp_binary <- function(
     sigma_b2 = sigma_b2,
     mu_b3 = mu_b3,
     sigma_b3 = sigma_b3,
+    bound = bound,
     link = link,
     w_prior = w_prior,
     longitudinal = longitudinal
   )
   class(mod) <- c("dreamer_exp_binary", "dreamer_binary")
   attr(mod, "type") <- "exponential"
+  rlang::inform("Note updated exponential model parameterization", 
```

Comment:
Please update message similar to that of the suggested EMAX model in other comment.